### ISSUE TITLE
This tx needs some help to cross the bridge

### ISSUE THREAD

**Issue by GriffGreen** [Giveth/giveth-dapps-v2#1042]
<img width="683" alt="Screen Shot 2022-06-30 at 4 15 42 PM" src="https://user-images.githubusercontent.com/14945613/176770892-f706322f-34fd-42c7-82ab-94c5d84a8e80.png">

**Comment by GriffGreen** [comment]
@danelsuga @aminlatifi @mohammadranjbarz

**Comment by mohammadranjbarz** [comment]
@GriffGreen  @danelsuga 
Thanks for reporting that, I handled that and the tx went
![Uploading Screen Shot 1401-04-10 at 19.20.56.png…]()

**Comment by danelsuga** [comment]
Thank you @mohammadranjbarz ! I guess this means it will take a couple days to get to my wallet?

**Comment by mohammadranjbarz** [comment]
@danelsuga 
Yeah, now your payment is `Approved` it would take few days to get `paid`

<img width="1917" alt="Screen Shot 1401-04-12 at 10 24 55" src="https://user-images.githubusercontent.com/9850545/177026892-8203e2be-3cda-4d68-a47f-220a3f21c72e.png">

**Comment by danelsuga** [comment]
(I don't have permissions to close the issue.)
